app/src/main/kotlin/com/mapzen/erasermap/presenter/MainPresenterImpl.kt

```diff
@@ -412,6 +487,53 @@ open class MainPresenterImpl(val mapzenLocation: MapzenLocation, val bus: Bus,
     }
   }
 
+  private fun prepareAndShowSearchResultsViews(features: List<Feature>?) {
+    prepareAndShowSearchResultsViews(features, 0)
+  }
+
+  /**
+   * When search results are selected this method handles adjusting all the views on the screen
+   * needed to properly display results. It makes the search results view visible, adjusts the
+   * attribution & find me btn as well as adds the search result pins to the map.
+   */
+  private fun prepareAndShowSearchResultsViews(features: List<Feature>?, activeIndex: Int) {
+    if (features == null) {
+      return
+    }
+
+    mainViewController?.hideReverseGeolocateResult()
+    mainViewController?.showSearchResultsView(features)
+    addSearchResultsToMap(features, activeIndex)
+    mainViewController?.layoutAttributionAboveSearchResults(features)
+    mainViewController?.layoutFindMeAboveSearchResults(features)
+    mainViewController?.toggleShowDebugSettings()
```

Comment:
You're right, it can be removed. It wasn't causing unexpected results because this method was never reached for the debug query term.

I've removed this call.